modules/ROOT/pages/editor-command-identifiers.adoc

```diff
@@ -321,12 +321,12 @@ The following command requires the xref:image.adoc[Image (`+image+`)] plugin.
 
 include::partial$commands/image-cmds.adoc[leveloffset=+3]
 
-[[imagetools]]
-==== Image Tools
+[[editimage]]
+==== Edit Image
 
-The following commands require the xref:imagetools.adoc[Image Tools (`+imagetools+`)] plugin.
+The following commands require the xref:editimage.adoc[Edit Image (`+editimage+`)] plugin.
 
-include::partial$commands/imagetools-cmds.adoc[leveloffset=+3]
+include::partial$commands/editimage-cmds.adoc[leveloffset=+3]
```

Comment:
This and the toc entry need moving to be in alphabetical order

- [ ] Move to the correct location and update the toc at the top of the page.